src/plugins/action_trim_responses/src/databaseMethods.ts

```diff
@@ -64,6 +64,38 @@ const databaseMethods = (DBConnect: any) => ({
       throw err
     }
   },
+  updateApplicationResponseTimestamps: async (responsesToUpdate: number[]) => {
+    const text = `UPDATE application_response
+      SET time_updated = current_timestamp
```

Comment:
I think we discussed for this to be matched to latest application/review status history ?

Reply:
Oh, I don't remember discussing that, sorry. That assumes the update status Action (to create a new status_history) would be ahead of the `trimResponses` Action. Is that a reasonable assumption? 
How about the changeStatus Action pass along the new status timestamp in its output, then we can make this Action use that timestamp if it exists, but write the current_timestamp if it doesn't?

Reply:
sure, or query for it in the update statement. up to you.

> That assumes the update status Action (to create a new status_history) would be ahead of the trimResponses

in the set `is visible` PR I'll make one empty template with 'core actions' (we are getting a few of them now, want to keep track of them)

Reply:
Yeah, I've been trying to keep a list of the core/default actions in #184

Reply:
Okay, changes to reflect the above: 65197f97

Haven't done extensive testing yet, but it passes the existing tests (which only deals with Level 1 reviews)